lib/zendesk-uploader/article-uploader.js

```diff
@@ -12,11 +12,13 @@ module.exports = class ArticleUploader extends Uploader {
     /**
      * Creates an ArticleUploader
      * @param {object} article - contains `meta` object with metadata and `section` reference to parent section
+     * @param {Config} config - global docpub configuraion
      * @param {object} [zendeskClient] - optional instance of the node-zendesk api client
      */
-    constructor(article, zendeskClient) {
-        super(article, zendeskClient);
+    constructor(article, config, zendeskClient) {
+        super(article, config, zendeskClient);
         this._article = article;
+        this._config = config;
```

Comment:
Lol, I don't know why I set it in 3 different places instead of just setting in basic uploader